### PR TITLE
Fix typos with codespell tool

### PR DESCRIPTION
**Pull request by ziemek99**
The proccess was manually supervised, so there's no need to worry that meaningful code has been corrupted. There still might be some other typos not detected by [`codespell`](https://github.com/codespell-project/codespell), so watch out.

### CODE REVIEW

File: deps.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ download_github() {
 main() {
   if git -C "${MYDIR}" rev-parse; then
     cat >&2 <<EOF
-Currenty directory is a git repository, downloading dependencies via git:
+Currently directory is a git repository, downloading dependencies via git:
```
Comment (veluca93) [inline deps.sh:62]: This should be current

Comment (ziemek99) [reply deps.sh]: Fixed

**Review by veluca93 — APPROVED** [review]

**Comment by ziemek99** [comment]
Why is **`PR / format (pull_request)`** check failing? Running `./ci.sh lint` on my machine went OK:
```
+ cmd_lint
+ merge_request_commits
+ git -C /home/ziemekz/libjxl-ziemek99 clang-format --binary clang-format --style=file --diff cf5cec5f405f5a2f74aad4ad5f8643c350ac4624 --
clang-format check OK
```

**Comment by ziemek99** [comment]
Fixed formatting, PTAL

**Review by deymo — APPROVED** [review]

**Pull request merged**